src/system/ExtensionsModule/Resources/views/ExtensionsInterface/footer.html.twig

```diff
@@ -0,0 +1 @@
+{# Not used #}
```

Comment:
admin footer *is* used ([here](https://github.com/zikula/core/pull/2648/files#diff-9727e1ebc679a82486081a5d50a496b7R2)) but I am not opposed this this - maybe add the module name and version?